openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -937,13 +937,27 @@
 - control_key: SC-17
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        While OpenShift auto-generates several PKIs to serve TLS on different
+        endpoints of the system, it is possible and necessary to configure a
+        custom PKI which allows external clients trust the endpoints.
+
+        The Ingress Operator is the component responsible for enabling external
+        access to OpenShift Container Platform cluster services. The aforementioned
+        operator creates an internal CA and issues a wildcard certificate that is
+        valid for applications under the .apps sub-domain. Both the web console
+        and CLI use this certificate as well. The certificate and key would need
+        to be replaced since a certificate coming from an approved provider is
+        needed. [1]
+
+        While the API Server certificates can't be replaced, it's possible to
+        add extra certificates that it will serve in case there's the need
+        for clients to have explicit trust for them. [2]
 
-        https://issues.redhat.com/browse/CMP-419
+        [1] https://docs.openshift.com/container-platform/4.7/security/certificates/replacing-default-ingress-certificate.html
+        [2] https://docs.openshift.com/container-platform/4.7/security/certificates/replacing-default-ingress-certificate.html
```

Comment:
The text itself is good, but I just wonder if both links are supposed to point to the same URL?

Reply:
Fixed.